### ISSUE TITLE
DELETE method doesn't work

### ISSUE THREAD

**Issue by davidgarciamt** [microsoftgraph/microsoft-graph-docs#17590]
The method is supposed to be deleting the onlineMeeting, and although it may do that, you can still join with the URL.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 56494a16-8774-5683-869a-859750bc71fb
* Version Independent ID: 4cbec427-5f5a-68d1-d4e5-1e7e5ae105f3
* Content: [Delete onlineMeeting - Microsoft Graph v1.0](https://docs.microsoft.com/en-us/graph/api/onlinemeeting-delete?view=graph-rest-1.0&tabs=http)
* Content Source: [api-reference/v1.0/api/onlinemeeting-delete.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/v1.0/api/onlinemeeting-delete.md)
* Product: **cloud-communications**
* Technology: **microsoft-graph**
* GitHub Login: @mkhribech
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by davidgarciamt** [comment]
No, i was getting the response with code "204", which means it has been deleted, but the thing is that even though it has been deleted, you can still access the onlineMeeting with the joinURL

**Comment by prashantsoni** [comment]
I have a similar isssue. 

Repro:
1.	Create meeting with OnlineMeeting create endpoint. Make a note of the Meeting URL. 
2.	Delete the meeting with delete endpoint
3.	Get meeting – responds not found
4.	Navigate to meeting url noted in step 1
5.	Get meeting - responds meeting found. 

Expected
Get call should respond meeting not found once deleted. 
Meeting URL should be disable. 

Actual 
On deleted meeting if the meeting url is opened the meeting urls are still active. 
Get call for the same meeting id returns found.
